Library/Formula/direnv.rb

```diff
@@ -23,8 +17,8 @@ def caveats; <<-EOS.undent
     At the END of your ~/.bashrc or ~/.zshrc, add the following line:
       eval "$(direnv hook $0)"
 
-    See the wiki for docs and examples:
-      https://github.com/zimbatm/direnv/wiki/
+    See the website for more docs and shell support:
+      http://www.direnv.net
```

Comment:
I would like to keep some form of instruction because without the shell integration direnv is not useful.